modules/main-rules/pages/attempt-to-rally-broken-detachments.adoc

```diff
@@ -4,14 +4,20 @@ If you can rally a Broken detachment then you will regain control so that you ca
 
 For each Broken detachment that you have, take a Rally test to see if they rally.
 
-.Rally test
-Roll a D6: If you score higher than the number of Blast markers on the detachment then you succeed -- remove the Broken condition marker from the detachment; otherwise, you fail -- leave the marker in place.
-
 [IMPORTANT]
 ====
-You cannot rally a detachment that has as many Blast markers (or more) as it has units remaining.
-
-You may be able to take the test for such detachments at the end of a future turn if you are able to remove enough Blast markers at step xref:roll-to-remove-blast-markers.adoc[].
+You may only attempt a Rally test when a Broken detachment has fewer Blast markers than the number of units it has remaining.
+{blank}[TODO: I think it'd be better to phrase this along the lines of "fewer Blast markers than its current detachment strength", to align with how we've defined this for {robust} units.]
```

Comment:
This is one of two instances like this that I've spotted — there may be more. On the one hand it'd be simpler and more intuitive to stick with units — but on the other hand, given the effect that _Robust_ has on detachment strength, I figure it'd be more consistent to base it on detachment strength so we don't have a disparity between half-strength for army morale and number of units for Rally tests...

Reply:
Agreed, it should be detachment strength. But...

As far as I can see, 'detachment strength' is currently an undefined term and only appears on the Robust page. I assume this will be expanded on in the battles section for Army Morale?

Reply:
**TL;DR**: We could skip over this query for now and just keep the unit-based text for the time being.

**Long version**:

Yes, to date we've only defined detachment strength as part of the the _Robust_ rules. I was thinking ahead a bit here, aiming to future-proof the text somewhat.

So, to think it through some more:

When we/I get around to adding drafts of the Battles book/Armies book content then 'detachment strength/half-strength' will necessarily feature as part of the explanation of the Army Morale rules/detachment card explanation. This concept already exists in the original rules, just in content we haven't gotten to yet because we've only completed a pass through the Core component, not Battles and Armies. (And of course we'll give it a more distinct definition.)

The experimental _Robust_ ability rules text explains how this ability should integrate with that concept — because this is a new idea that didn't exist in the source and so wasn't accounted for.

The IMPORTANT note for the Rally test a question of how to apply _Robust_ in this context — and it seems like the the obvious and most consistent solution is to unify with the detachment strength concept and use the same terminology.

But, given that the _Robust_ ability is still experimental at this point in time, perhaps the best path for the moment is to:

1. Stick with the unit-based phrasing here for the moment.
2. Tweak/expand the experimental _Robust_ rules text to make the Rally test situation plain as well. Or at least add a TODO note there so that we have a reminder and players have a pointer. (Either way, we should probably create an issue so we don't forget, and then a branch to implement it.)
3. Revisit if/when we add the other content and/or make _Robust_ 'official' in future, and integrate as appropriate.

What do you reckon?

Reply:
I'd be happy with 2. It was more a comment that we haven't defined detachment strength yet, than that something needed changing.

Reply:
Just to be clear, I meant that as a list of three actions to take in combination, not three separate options. I've now created issue #41 to this effect so we don't lose sight of it, and thus we can clean up **this** PR a bit:

```suggestion
```